doc/mk_sdcard_image.sh

```diff
@@ -71,19 +72,29 @@ partition_image()
 {
 	print_step "Make partitions"
 
-	sudo parted -s $1 mklabel msdos
+	sudo parted -s $1 mklabel msdos || true
 
-	sudo parted -s $1 mkpart primary ext4 1MiB 257MiB
-	sudo parted -s $1 mkpart primary ext4 257MiB 2257MiB
-	sudo parted -s $1 mkpart primary ext4 2257MiB 3415MiB
-	sudo parted -s $1 mkpart extended 3415MiB 100% 
-	sudo parted -s $1 mkpart logical ext4 3416MiB 5515MiB
-	sudo parted -s $1 mkpart logical ext4 5516MiB 5773MiB
-	sudo parted -s $1 mkpart logical ext4 5774MiB 5775MiB
-	sudo parted -s $1 mkpart logical ext4 5776MiB 100%
+	sudo parted -s $1 mkpart primary ext4 1MiB 257MiB || true
+	sudo parted -s $1 mkpart primary ext4 257MiB 2257MiB || true
+	sudo parted -s $1 mkpart primary ext4 2257MiB 3415MiB || true
+	sudo parted -s $1 mkpart primary 3415MiB 100% || true
 	sudo parted $1 print
```

Comment:
don't you want to put this below after all is done?

Reply:
Each parted command can generate output like this: Error: Partition(s) 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64 on /dev/sdc4 have been written, but we have been unable to inform the kernel of the change, probably because it/they are in use.  As a result, the old partition(s) will remain in use.  You should reboot now before making further changes.

So we need to guard each command.